setup.py

```diff
@@ -24,7 +24,7 @@
     description="Library to handle signal data and perform signal processing computations",
     entry_points={"console_scripts": ["signalworks=signalworks.cli:main"]},
     install_requires=["numpy", "scipy", "numba==0.42.0", "llvmlite==0.27.0"],
-    extras_require=["pyedflib", "openxdf"],
+    extras_require={"label": ["pyedflib", "openxdf"]},
```

Comment:
Add `soundfile` as an optional dependency here; under `robust_audio` tag

so it should be

```python
extras_require = {"label": ["pyedflib", "openxdf"], "robust_audio": ["soundfile"]}
```